src/shared-components/ProfileForm/hooks/useProfileFormState.ts

```
@@ -0,0 +1,101 @@
+  const initializeFormState = (userData: any) => {
+    if (userData?.user) {
+      setFormState(userData.user);
+      originalImageState.current = userData.user?.avatarURL || '';
+    }
+  };
+
+  const handleFieldChange = (fieldName: string, value: string) => {
+    if (fieldName === 'password' && value) {
+      if (!validatePassword(value)) {
+        NotificationToast.error(
+          tCommon('passwordLengthRequirement', { length: 8 }) as string
```

ESLint

[error] Insert `,`

(prettier/prettier)